src/geometry/PolyUtil.js

```diff
@@ -95,5 +107,23 @@ export function polygonCenter(latlngs, crs) {
 	} else {
 		center = [x / area, y / area];
 	}
-	return crs.unproject(toPoint(center));
+
+	var latlngCenter = crs.unproject(toPoint(center));
+	return toLatLng([latlngCenter.lat + centroidLatLng.lat, latlngCenter.lng + centroidLatLng.lng]);
+}
+
+/* @function centroid(latlngs: LatLng[]): LatLng
+ * Returns the 'center of mass' of the passed LatLngs.
+ */
+export function centroid(coords) {
+	var latSum = 0;
+	var lngSum = 0;
+	var len = 0;
+	coords.forEach(function (coord) {

```

Comment:
I believe we're avoiding using `.forEach()` in v1 as it's not supported by all the browsers. So this should probably be a classic loop.

Reply:
makes sense, I will replace that